api/v1alpha1/recipe_types.go

```diff
@@ -22,14 +22,19 @@ type RecipeSpec struct {
 	//+listType=map
 	//+listMapKey=name
 	Groups []*Group `json:"groups"`
+	// Volumes to protect from disaster
+	//+optional
+	Volumes *Group `json:"volumes"`
 	// List of one or multiple hooks
 	//+listType=map
 	//+listMapKey=name
 	Hooks []*Hook `json:"hooks,omitempty"`
-	// Workflow is the sequence of actions to take
-	//+listType=map
-	//+listMapKey=name
-	Workflows []*Workflow `json:"workflows"`
+	// ProtectWorkflow is the sequence of actions to take to protect from disaster
+	//+optional
+	ProtectWorkflow *Workflow `json:"protectWorkflow"`
```

Comment:
In general when referring to DR, I thought we were using "capture", rather than "protect".

Reply:
Yes, this workflow is about ordering the reading part of the "backup" and the writing to a protected location part can actually happen later and its details are not specified in the workflow, so I think capture is a better term.